multi.html

```diff
@@ -6,10 +6,12 @@
     <title>TANX</title>
```

Comment:
we might as point to this, instead of 
https://github.com/cvan/moz-tanx/blob/master/public/index.html